ciao-scheduler/scheduler.go

```diff
@@ -697,10 +709,10 @@ func startWorkload(sched *ssntpSchedulerServer, controllerUUID string, payload [
 
 	var targetNode *nodeStat
 
-	if workload.networkNode == 0 {
-		targetNode = pickComputeNode(sched, controllerUUID, &workload)
-	} else { //workload.network_node == 1
+	if workload.networkNode {
 		targetNode = sched.pickNetworkNode(controllerUUID, &workload)
+	} else { //workload.network_node == false
```

Comment:
This is related to unit test.  Until now the pickNetworkNode() was so trivial that I didn't export it in export_test.go for use in scheduler_internal_test.go.  The function route is required for the export to work.  So pickNetworkNode() will also shift to being a function instead of a method so that it can see isolated "go test" and "go bench" based testing yet also remain unexported (ie: we've deliberately chosen to not have an arbitrarily pluggable scheduler).